### PR TITLE
Add osu! World Cup #3 article

### PR DESCRIPTION
**Pull request by TPGPL**
Related to #350 

- No Match Results because Stat sheet is dead
- No links in Participants table because it's not a priority i guess and lots of players changed their nicknames

---

### CODE REVIEW

File: wiki/Tournaments/OWC/3/en.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+| Semifinals | 2013-01-11/2013-01-13 |
+| Finals | 2013-01-18/2013-01-20 |
+
+## Prizes
+
+In every world cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place")   | 6 months of osu!supporter tag, unique profile badge and a T-Shirt |
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/3/en.md]: Remove the double space.

`and a` -> `,`

File: wiki/Tournaments/OWC/3/en.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+- HardRock
+  - [Hommarju feat. R.Cena - Chousai Kenbo Sengen (galvenize) \[Insane\]](http://osu.ppy.sh/beatmaps/99342)
+  - [Rohi - Kodoku Egoism (NatsumeRin) \[Rin\]](http://osu.ppy.sh/beatmaps/196673)
+  - [Kagamine Rin - Black Rebel (val0108) \[0108 Rebel\]](http://osu.ppy.sh/beatmaps/109301)
+- DoubleTime
+  - [Touyama Nao - Happy Crescent (Sumisola) \[Insane\]](http://osu.ppy.sh/beatmaps/94634)
+  - [Matchbox Twenty - How Far We've Come (Sushi) \[Insane\]](http://osu.ppy.sh/beatmaps/104117)
+  - [doriko feat. Hatsune Miku - Paradise Cage (KanbeKotori) \[Insane\]](http://osu.ppy.sh/beatmaps/80384)
+- Tiebreaker
+  - [HujuniseikouyuuP - Talent Shredder (val0108) \[Lesjuh style\]](http://osu.ppy.sh/beatmaps/153857)
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/3/en.md]: Bold the tiebreaker.

File: wiki/Tournaments/OWC/3/en.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/102985)
+- [Livestream](https://www.twitch.tv/osuworldcup)
+
+-------------------------
+
+## Participants
+
+|  | Country | Members |
+| :-: | :-: | :-- |
+| ![][flag_AR] | Argentina | **Metro**, Darksonic, Glazbom, Fabi, Hernan, Salvage, RocknRolla, druidxd |
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/3/en.md]: Bold the countries.

File: wiki/Tournaments/OWC/3/en.md [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+  - [Hatsune Miku - Kimi ni Uso (KanaRin) \[Kana\]](http://osu.ppy.sh/beatmaps/87645)
+  - [yonji - Waruku nai ne (Reiji-RJ) \[Hard\]](http://osu.ppy.sh/beatmaps/109757)
+- Tiebreaker
+  - [Hatsune Miku - Cruel Clocks (Nyquill) \[Full Revolution\]](http://osu.ppy.sh/beatmaps/148307)
+
+-----------------------
+
+## Ruleset
+
+## Tournament Registration
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/3/en.md]: Fix the header level.

**Comment by fajar13k** [comment]
Shouldn't we escape `#3`?

**Comment by Joehuu** [comment]
I don't think `#` does anything.

File: wiki/Tournaments/OWC/3/en.md [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+
+## Organisation
+
+The osu! World Cup #3 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management | ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366), ![][flag_PL] [fartownik](https://osu.ppy.sh/users/56917) |
+| Referees | ![][flag_BG] [Flanster](https://osu.ppy.sh/users/447818), ![][flag_CA] [Xakyrie](https://osu.ppy.sh/users/684730), ![][flag_CA] [Haruka](https://osu.ppy.sh/users/686460), ![][flag_FR] [Cyclohexane](https://osu.ppy.sh/users/116078)
+| Map Selectors | ![][flag_IT] [Chewin](https://osu.ppy.sh/users/617323), ![][flag_HR] [lolcubes](https://osu.ppy.sh/users/578401), ![][flag_KR] ykcarrot, ![][flag_AR] [Wishy](https://osu.ppy.sh/users/495477), ![][flag_NO] [kriers](https://osu.ppy.sh/users/333241)
```
Comment (cl8n) [inline wiki/Tournaments/OWC/3/en.md]: ykcarrot is https://osu.ppy.sh/users/70730

File: wiki/Tournaments/OWC/3/en.md [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+5. Alternately each captain selects a map out of the mappool. Captains should agree to each other who will select the first map. If no agreement can't be reached, blue team starts selecting.
+   - The captains can select maps out of the NoMod bracket unlimited.
+   - Selection out of mod-specific brackets is limited. Each captain can only select one map from each mod-restricted bracket during match. So each team can pick 1 HardRock, 1 Hidden and 1 DoubleTime map during match.
+   - In case of tie, the tiebreaker map must be played.
+6. The settings of the multiplayer room will be: osu! standard mode, `Team VS` mode and win condition: score.
+7. You are free to set 'No Video' mod. Also, you are free to use the Funspoiler options.
+8. The country with best total score on the map wins.
+9. The teams can exchange only one player per map during matches.
+
+### Other Rules
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/3/en.md]: `Other Rules` -> `Tournament Rules`?

**Comment by Nivalyx** [comment]
The group stages aren't actually split by "rounds" (as they are technically all in the same phase) but rather by "weeks" so it might be better to change "Group Stage (Round x)" to "Group Stage (Week x") perhaps?

File: wiki/Tournaments/OWC/3/en.md (hunk 6 above)
Comment (TPGPL) [reply wiki/Tournaments/OWC/3/en.md]: i would leave it like that, it was the same way in the forum thread

File: wiki/Tournaments/OWC/3/en.md [hunk 7]
```diff
@@ -0,0 +1,370 @@
...
+| ![][flag_RU] | **Russia** | **Kert**, Rain, Rost94, cr1m, JuZu, NekoMimiMode, funshine, Shandro |
+| ![][flag_SG] | **Singapore** | **Bardiche\_Z**, Onozuka-Komachi, rainfall1997, WTPhox, jonchua, Renegate |
+| ![][flag_SE] | **Sweden** | **Failed**, Nana Tan, Gyuunyu, Cloudio-San, Kawaii, Slizzer95 |
+| ![][flag_TW] | **Taiwan** | **Uan**, Rucker, SnowWhite, ms0080345, Small K, Small AB, melin85116, Oxob |
+| ![][flag_TH] | **Thailand** | **NonxE**, Frostmourne, 0OoMickeyoO0, blackspell, bufo, nick4010, termerys, phuditj |
+| ![][flag_TR] | **Turkey** | **Pure Force**, Shadowa, S4LeagueSexsiman, -Lvs, HellDawn, XavierJones, BonkuraChan |
+| ![][flag_UA] | **Ukraine** | **Yoru-SAN**, rockleejkooo, blednak, Granje, Gorlum, mrgnstrn |
+| ![][flag_GB] | **United Kingdom** | **jesus1412**, heat, R a h a r u, Vapor, Eastern Aristotle, Raiku |
+| ![][flag_US] | **United States** | **Derekku**, Kyou-kun, SapphireGhost, Kaoru, JAKANYAN, Kyonko Hizara, Sync, pielak213- |
+| ![][flag_UG] | **Uruguay** | **maay**, AlrdyExists, GonixZ, HeCAtExZz, sevas-tra, Snepif, W a r g, Z e o n |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/3/en.md]: This is not the Uruguay flag, should be UY

**Pull request merged**